ts/screens/wallet/FirstTransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,245 @@
+/**
+ * This screen shows the transaction details.
+ * It should occur after the transaction identification by qr scanner or manual procedure.
+ * TODO:
+ * - integrate contextual help
+ *    https://www.pivotaltracker.com/n/projects/2048617/stories/158108270
+ *
+ * - check availability of displayed data. Define optional data and implement their rendering as preferred
+ *
+ *
+ */
+
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  H3,
+  Icon,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { Col, Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { WalletAPI } from "../../api/wallet/wallet-api";
+import { WalletStyles } from "../../components/styles/wallet";
+import AppHeader from "../../components/ui/AppHeader";
+import PaymentSummaryComponent from "../../components/wallet/paymentSummary/PaymentSummaryComponent";
+import UpdatedPaymentSummaryComponent from "../../components/wallet/paymentSummary/UpdatedPaymentSummaryComponent";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+import {
+  NotifiedTransaction,
+  TransactionEntity,
+  TransactionSubject
+} from "../../types/wallet";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+const styles = StyleSheet.create({
+  padded: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  }
+});
+
+const transactionDetails: Readonly<
+  NotifiedTransaction
+> = WalletAPI.getNotifiedTransaction();
+const entityDetails: Readonly<
+  TransactionEntity
+> = WalletAPI.getTransactionEntity();
+const subjectDetails: Readonly<
+  TransactionSubject
+> = WalletAPI.getTransactionSubject();
+
+export class FirstTransactionSummaryScreen extends React.Component<
+  Props,
+  never
+> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  private goBack() {
+    this.props.navigation.goBack();
+  }
+
+  /**
+   * Depending on the comparison between the amount on the notice and the amount saved remotely by the lender
+   * it will be displayed a different component. If the values differ, then the user can display both the value
+   * and a brief exmplanation.
+   */
+  private isAmountUpdated() {
+    if (
+      transactionDetails.currentAmount.match(
```

Comment:
I guess this could be:

```ts
private isAmountUpdated() {
   return transactionDetails.currentAmount !==  transactionDetails.notifiedAmount
}
```

Reply:
they was strings. anyway, I updated to numbers in the next commit

Reply:
`===` (or `!==`) works for string and numbers apart some oddities with floats ie.
`0.1+0.2==0.3` --> false